docs/releases/1.8.txt

```diff
@@ -990,6 +1016,126 @@ Miscellaneous
 Features deprecated in 1.8
 ==========================
 
+.. _model_meta_upgrade:
```

Comment:
I've moved this to a section at the end of the Meta docs, because it didn't seem substantial enough to warrant an entirely new page, and that page already contains references to the fact that there was an "old" API.